Gruntfile.js

```diff
@@ -361,6 +362,7 @@ module.exports = function (grunt) {
       'copy:generate-in-translation',
       'propertiesToJSON',
       'copy:app-to-target',
+      'exec:pseudo-loc', // generates pseudo-loc
```

Comment:
This is very similar to what @nikhilvenkatraman-okta and I are thinking. Yet, instead of hooking into our lint process (we'll use our custom ESLint rules ([here](https://github.com/okta/okta-ui/blob/master/scripts/eslint-plugin-okta-ui/docs/rules/no-unlocalized-text.md) and [here](https://github.com/okta/okta-ui/blob/master/scripts/eslint-plugin-okta-ui/docs/rules/no-unlocalized-text-in-templates.md)) for that) we'll plug into our testcafe testing framework. This, along with making `ok_PL` the **default** development locale will help us catch English leaks at dev time.

Reply:
Commented this out with a JIRA follow-up: OKTA-379995